core/chains/evm/txmgr/broadcaster.go

```diff
@@ -94,7 +109,8 @@ type EthBroadcaster[
 	R txmgrtypes.ChainReceipt[TX_HASH, BLOCK_HASH],
 	SEQ txmgrtypes.Sequence,
 	FEE txmgrtypes.Fee,
-	ADD any,
+	ADD any, // additional parameter used in tx construction
+	UNIT txmgrtypes.Unit, // base fee unit type
```

Comment:
Does it make sense to rename UNIT to FEE_UNIT everywhere?
It is hard for reader to understand what a generic UNIT type means, without searching for the comment about this type.